upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -1,109 +1,160 @@
+    public array $pathsToCheck = [];
+
+    /** @var string Session key to track cache cleanup completion */
+    protected string $cleanCacheKey = 'cache-cleaned-270';
 
     /**
-     * __construct
+     * @param XoopsMySQLDatabase $db      database connection
+     * @param UpgradeControl     $control upgrade control instance
      */
-    public function __construct()
+    public function __construct(XoopsMySQLDatabase $db, UpgradeControl $control)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)